maryana_khoptyak/Task_1/test_get_max_value.py

```diff
@@ -0,0 +1,44 @@
+EMPTY_SEQUENCE = []
+ONLY_ONE_ELEM_NUM = ["15"]
+ONLY_ONE_ELEM_CHR = ["some  long text goes here"]
+ONLY_CHR = ["one", "two", "ten", "some  long text goes here"]
+ONLY_NUMBERS = ["1", "2", "10", "15", "-2", "16.9"]
+DATA = ["one", "1", "two", "2", "ten", "10", "15", "-2", "some  long text goes here", "16.9", "&"]
+
+
+def get_max_value(data):
+    """ Function which returns a maximum integer value from list of strings """
+    sorted_nums_list = []
+    for value in data:
+        try:
+            sorted_nums_list.append(float(value))
+        except ValueError:
+            pass
+    if sorted_nums_list:
```

Comment:
It might be replaced with ternary operator:

    return max(sorted_nums_list) if sorted_nums_list else None

Reply:
Replaced.